sqlalchemy_bigquery/base.py

```diff
@@ -963,7 +971,10 @@ def _get_table(self, connection, table_name, schema=None):
             raise NoSuchTableError(table_name)
         return table
 
-    def has_table(self, connection, table_name, schema=None):
+    def has_table(self, connection, table_name, schema=None, **kw):
+        """
+        No kw are supported
```

Comment:
If we are gonna make and edit and have a docstring here, I would recommend that we take the time to create a useful docstring that defines the arguments and return values and clarifies what is meant by 'no kw are supported'.

Reply:
Seems to be the pattern followd by sqlalchemy codebase too:

Oracle seems to be only dialect implementation that supports a `**kw` approach.

https://github.com/sqlalchemy/sqlalchemy/blob/d8cab0694e6dc9f7b7fd02a4b5a8ae6bbb2e896e/lib/sqlalchemy/dialects/oracle/base.py#L1617

Reply:
Added a docstring for this function, however we should eventually add useful docstrings to all functions in this file for readability.